src/track/beats.cpp

```diff
@@ -746,83 +781,85 @@ Bpm BeatsInternal::getBpmAroundPosition(FramePos curFrame, int n) const {
     stopBeat.set_frame_position(upper_bound.getValue());
     return calculateBpm(startBeat, stopBeat);
 }
-TimeSignature BeatsInternal::getSignature(FramePos frame) const {
+
+TimeSignature BeatsInternal::getSignature(int beatIndex) const {
     if (!isValid()) {
```

Comment:
What is the meaning of an invalid time signature? A time signature with 0 as one of the numbers?